source/attributes/data-lighting.md

```diff
@@ -0,0 +1,22 @@
+# data-lighting
+
+```html
+<mr-app data-lighting="enabled=true; shadows:true; color:#faa; intensity:0.2; radius:4;"> ... </mr-app>
```

Comment:
this is a bit weird: `enabled=true; shadows:true;`
Is that a mistake or an unfortunate code thingie we have to deal with?
`enabled:true; shadows:true;` would be nicer

Reply:
good catch - thankfully i think that's just a bad typo on my part - i'll clean that up ~